src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
 // write your code here
+function success() {
+  const message = document.createElement('div');
+  message.textContent = 'Promise was resolved!';
+  document.querySelector('body').appendChild(message);
+}
+
+function error() {
+  const message = document.createElement('div');
+  message.textContent = 'Promise was rejected!';
+  document.querySelector('body').appendChild(message);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  const button = document.querySelector('h1');
```

Comment:
The selector `document.querySelector('h1')` is used to select a button. Ensure that there is an `h1` element in your HTML that acts as a button. If not, you should use the correct selector for the button element.